ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/StructLog.java

```diff
@@ -36,7 +37,7 @@
   private final int pc;
   private final String[] stack;
   private final Object storage;
-  private final String reason;
+  private final BytesValue reason;
```

Comment:
I think this should be a String here, but calculated from the `BytesValue` with `BytesValue.toUnprefixedString`.  That would then match how the stack and return values are all formatted.